client/odie/message/thumbs-icons.tsx

```diff
@@ -0,0 +1,31 @@
+export const ThumbsUpIcon = ( { className }: { className?: string } ) => (
+	<svg
+		xmlns="http://www.w3.org/2000/svg"
+		width="24"
+		height="24"
+		viewBox="0 0 24 24"
+		className={ className }
+	>
+		<path
+			fill-rule="evenodd"
+			clip-rule="evenodd"
```

Comment:
Warnings. Need camelcase for these (fill-rule too)
![image](https://github.com/Automattic/wp-calypso/assets/7129409/bf065df5-81d2-4139-a682-a655e333566d)